### DISCUSSION TITLE
Installation of iso on VM

### DISCUSSION THREAD

**Discussion by cstrong84** [Security-Onion-Solutions/securityonion#11163] [General]
I am attempting to install onion security with a validated gpg key to make sure the image is good.  However, no matter which of the options I use to attempt to install onion from the ISO, I keep receiving the same error:
The following error occurred while installing the payload.  This is a fatal error and installation will be aborted.

DNS error: Error unpacking rpm package kernel-uek-modules-5.15.0-103.114.4.el9uek.x86_64

Press ENTER to exit.  

Is there a way to fix this in order to get it installed?

**Comment by dougburks** [comment]
> DNS error: Error unpacking rpm package kernel-uek-modules-5.15.0-103.114.4.el9uek.x86_64

Based on the kernel version, it looks like you're installing Security Onion 2.4. If that's the case, please create a new discussion at https://github.com/Security-Onion-Solutions/securityonion/discussions/categories/2-4 as that will provide more information so that we can help you faster.
